Sources/CoreCommands/Options.swift

```diff
@@ -524,6 +524,10 @@ public struct LinkerOptions: ParsableArguments {
     /// If should link the Swift stdlib statically.
     @Flag(name: .customLong("static-swift-stdlib"), inversion: .prefixedNo, help: "Link Swift stdlib statically")
     public var shouldLinkStaticSwiftStdlib: Bool = false
+
+    /// Disables adding $ORIGIN/@loader_path to the rpath, useful when deploying
+    @Flag(name: .customLong("disable-local-rpath"), help: "Disable adding $ORIGIN/@loader_path to the rpath by default")
+    public var shouldDisableLocalRpath: Bool = false
```

Comment:
I don't care what the flag name is, feel free to suggest another.

Reply:
I would call it "origin" instead of "local"

Reply:
I considered that originally but used this broader term when I disabled the Darwin `@loader_path` too. Do you want me to keep that disabled for Darwin and change this flag and variable name to `origin`? I have no preference on these two aspects of this pull.